pkg/security/podsecuritypolicy/provider.go

```diff
@@ -295,6 +299,10 @@ func (s *simpleProvider) ValidatePod(pod *api.Pod) field.ErrorList {
 		}
 	}
 
+	if s.psp.Spec.RuntimeClass != nil && len(s.psp.Spec.RuntimeClass.AllowedRuntimeClassNames) > 0 {
```

Comment:
change to treat "*" as allow all and len(AllowedRuntimeClassNames) == 0 as allow none